### PR TITLE
Use libram free run ActionSource

### PR DESCRIPTION
**Pull request by Rinn**
Free kills will come in another PR

### CODE REVIEW

File: src/fights.ts [hunk 1]
```diff
@@ -578,20 +579,18 @@ class FreeRunFight extends FreeFight {
     if (!this.available()) return;
     if ((this.options.cost ? this.options.cost() : 0) > get("garbo_valueOfFreeFight", 2000)) return;
     while (this.available()) {
-      const runSource = findRun(this.options.familiar ? false : true);
+      const runSource = tryFindFreeRun(this.options.familiar ? { noFamiliar: () => true } : {});
```
Comment (horrible-little-slime) [inline src/fights.ts:582]: is there a reason it's not `{ noFamiliar: () => this.options.familiar !== undefined }`?

File: src/lib.ts [hunk 2]
```diff
@@ -212,199 +204,13 @@ export function tryFeast(familiar: Familiar): void {
...
-    (a, b) => mallPrice(a) - mallPrice(b)
-  )[0];
-  return new FreeRun(
-    "LTB",
-    () => retrieveItem(freeRunItem),
-    Macro.item(freeRunItem),
-    new Requirement([], {}),
-    () => retrieveItem(freeRunItem)
-  );
+export const ltbRun: () => ActionSource = () => {
```
Comment (horrible-little-slime) [inline src/lib.ts:207]: It feels like the new ActionSource system obsoletes this, but also I get that we probably don't want to do too much surgery on the grape all at once

**Review by horrible-little-slime — APPROVED** [review]
If this is tested, it looks good to me. Great work!

File: src/lib.ts (hunk 2 above)
Comment (Rinn) [reply src/lib.ts:207]: yes that's why i left it, i'm gonna do another pass

File: src/fights.ts (hunk 1 above)
Comment (Rinn) [reply src/fights.ts:582]: sure

**Pull request merged**